### DISCUSSION TITLE
Plan to upgrade electron dependencies

### DISCUSSION THREAD

**Discussion by peaceabuya** [eclipse-theia/theia#12849] [General] [answered]
I noticed that the Electron version 23.0.0 series is no longer being supported (https://www.electronjs.org/docs/latest/tutorial/electron-timelines#end-of-life)

Is there any plan to upgrade electron dependencies in the Theia project?

Thanks

**Comment by vince-fugnitto** [comment]
@peaceabuya thank you for the discussion, in general we try to follow the `electron` version that vscode uses. 
It seems that they bumped to `25.x.y` just two weeks ago (https://github.com/microsoft/vscode/pull/188268#issue-1811840991), so we can certainly upgrade.

**Reply by peaceabuya** [reply]
Great, thanks for the prompt answer. Any timeline for the upgrade?

**Reply by vince-fugnitto** [reply]
Nothing has been planned yet.

**Reply by vince-fugnitto** [reply, marked as answer]
I've created the following issue to track the upgrade (https://github.com/eclipse-theia/theia/issues/12855).
